typescript-testing-examples/simple-test/tests/collection_helper.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseRun, Run } from "langsmith/schemas";
+import { BaseTracer } from "langchain/callbacks";
+
+// Todo: Remove in next JS release
+export class RunCollectorCallbackHandler extends BaseTracer {
+  name = "run_collector";
+  exampleId?: string;
+  tracedRuns: Run[];
+
+  constructor({ exampleId }: { exampleId?: string } = {}) {
+    super();
+    this.exampleId = exampleId;
+    this.tracedRuns = [];
+  }
+
+  protected async persistRun(run: BaseRun): Promise<void> {
+    const run_ = { ...run } as Run;
```

Comment:
Is the cast necessary? I think you can just do:
`const run_: Run = { ...run };`

Reply:
Oh probably! Need to type as run instaed of baserun but that probably works